docs/developing.md

```diff
@@ -71,81 +71,81 @@ pytest -v --full-trace lib/rucio/tests/test_replica.py:TestReplicaCore.test_dele
 
 Again run the containers using docker-compose:
 
-```
+```bash
 docker-compose --file etc/docker/dev/docker-compose-storage.yml up -d
 ```
 
 This should show you a few more running containers: the Rucio server, the PostgreSQL database, FTS and its associated MySQL database, the Graphite monitoring, and three XrootD storage servers.
 
 With this container you can upload and download data to/from the storage and submit data transfers. To set this up, add the -r option to the setup.
-
-> tools/run_tests_docker.sh -ir
-
+```bash
+tools/run_tests_docker.sh -ir
+```
 This creates a few random files and uploads them, creates a few datasets and containers, and requests a replication rule for the container, which starts in state REPLICATING. To demonstrate the transfer capability, the daemons can be run in single-execution mode in order:
 
-> rucio rule-info <rule-id>
-
-> rucio-conveyor-submitter –run-once
-> rucio-conveyor-poller –run-once –older-than 0
-> rucio-conveyor-finisher –run-once
-
-> rucio rule-info <rule-id>
+```bash
+rucio rule-info<rule-id>
+rucio-conveyor-submitter –run-once
+rucio-conveyor-poller –run-once –older-than 0
+rucio-conveyor-finisher –run-once
+rucio rule-info <rule-id>
```

Comment:
Please add a prefix like a `$` sign when there are multiple commands in a code-block